src/main/java/de/hysky/skyblocker/skyblock/dungeon/Trivia.java

```diff
@@ -42,14 +46,18 @@ public boolean onMatch(Text message, Matcher matcher) {
     }
 
     private void updateSolutions(String question) {
-        String trimmedQuestion = question.trim();
-        if (trimmedQuestion.equals("What SkyBlock year is it?")) {
-            long currentTime = System.currentTimeMillis() / 1000L;
-            long diff = currentTime - 1560276000;
-            int year = (int) (diff / 446400 + 1);
-            solutions = Collections.singletonList("Year " + year);
-        } else {
-            solutions = Arrays.asList(answers.get(trimmedQuestion));
+        try {
+            String trimmedQuestion = question.trim();
+            if (trimmedQuestion.equals("What SkyBlock year is it?")) {
+                long currentTime = System.currentTimeMillis() / 1000L;
+                long diff = currentTime - 1560276000;
+                int year = (int) (diff / 446400 + 1);
+                solutions = Collections.singletonList("Year " + year);
+            } else {
+                solutions = Arrays.asList(answers.get(trimmedQuestion));
```

Comment:
You can just put a null check for `answers.get(trimmedQuestion)`

Reply:
> You can just put a null check for `answers.get(trimmedQuestion)`

Added a null check but I'm keeping the try catch because if something does happen it should not cause the game to momentarily freeze